apps/site/lib/site_web/controllers/schedule/line.ex

```diff
@@ -39,16 +39,33 @@ defmodule SiteWeb.ScheduleController.Line do
   end
 
   def do_call(
-        %Conn{assigns: %{route: %Route{} = route, direction_id: direction_id}} = conn,
+        %Conn{
+          assigns: %{
+            route: %Route{} = route,
+            direction_id: direction_id
+          }
+        } = conn,
         args
       ) do
+    schedule_direction = conn.query_params["schedule_direction"] || %{}
+
+    direction_id_value =
+      case schedule_direction["direction_id"] do
```

Comment:
@alicia-mbta something has occurred to me, and this one's on me, I should have thought of this earlier. We need to make this backwards-compatible so that if someone has a bookmark saved using the old `direction_id` and `variant` parameters, it should honor those.

Reply:
@phildarnowsky @alicia-mbta are you thinking to achieve this via a redirect?

Reply:
Yes, @phildarnowsky has explained this to me. The changes for that are in file `dotcom/apps/site/lib/site_web/controllers/schedule/line.ex`